plugins/GSdx/Renderers/Common/GSOsdManager.cpp

```diff
@@ -405,12 +416,74 @@ size_t GSOsdManager::GeneratePrimitives(GSVertexPT1* dst, size_t count) {
 		for(const auto &pair : m_monitor) {
 			if((pair.first.size() + pair.second.size()) * 6 > count - drawn) break;
 
-			// Calculate where to start rendering from by taking the right most position 1.0
-			// and subtracting (going left) 8 scaled pixels for a margin, then subtracting
-			// the size of the longest key and subtracting a scaled space and finally
-			// subtracting the longest value
-			float x = 1.0f - 8 * (2.0f/m_real_size.x) - first_max - m_char_info[' '].ax * (2.0f/m_real_size.x) - second_max;
-			float y = -1.0f + ((m_size+2)*(2.0f/m_real_size.y)) * line++;
+			float x, y;
```

Comment:
Should be initialized.

Reply:
Same story here, it will always be initialized because of the default case below.